File: Lista1/Java/ex_11.java
import java.util.Scanner;
public class ex_11
{
	public static void main(String[] args) {
	    Scanner sc = new Scanner(System.in);
		int soma_idade = 0;
        int media_idade = 0;
		int qtd_pessoas = 0;
	    for (int c = 1; c<=7; c++) {
	      System.out.println("Digite a idade da " + c + "° pessoa: ");
	      int idade = sc.nextInt();
          soma_idade += idade;
          System.out.println("Digite a altura da " + c + "° pessoa: ");
          double peso = sc.nextDouble();
		  if (peso > 90) {
			qtd_pessoas ++;
		  }
	    }
        media_idade = soma_idade/7;
        
		System.out.println("A quantidade de pessoas com mais de 90kg é : " + qtd_pessoas);
        System.out.println("A média da idade é : " + media_idade);
	}
}

File: Lista1/Python/ex_11.py
cont_peso = 0
soma_idade = 0
media_idade = 0
for c in range(1,8):
    idade = int(input(f'Digite a idade da {c}° pessoa: '))
    soma_idade += idade
    peso = float(input(f'Digite o peso da {c}° pessoa: '))
    if peso > 90:
        cont_peso += 1
media_idade = soma_idade/7
print(f'A quantidade de pessoas com peso maior que 90kg é: {cont_peso}')
print(f'A media da idade é: {media_idade}')